indico/modules/events/ical.py

```diff
@@ -130,7 +132,7 @@ def events_to_ical(events, user=None, scope=None):
     calendar.add('prodid', '-//CERN//INDICO//EN')
 
     for event in events:
-        if not event.can_access(user):
+        if not event.can_access(user) and not skip_access_check:
```

Comment:
```suggestion
        if not skip_access_check and not event.can_access(user):
```

no need to check if if the result is ignored